README.md

```diff
@@ -236,40 +236,76 @@ Check out [contributing guide](https://github.com/GoogleChrome/puppeteer/blob/ma
 
 # FAQ
 
-#### Q: Which Chromium version does Puppeteer use?
-
-Look for `chromium_revision` in [package.json](https://github.com/GoogleChrome/puppeteer/blob/master/package.json).
```

Comment:
Think we should keep this one. "Which Chromium version does Puppeteer use?" so people know how to lookup the version.

Reply:
Brought it back right after "Why puppeteer vXXX doesn't work with chromium vYYY"